pandas/_libs/tslibs/parsing.pyx

```diff
@@ -44,13 +50,73 @@ class DateParseError(ValueError):
 
 _DEFAULT_DATETIME = datetime(1, 1, 1).replace(hour=0, minute=0,
                                               second=0, microsecond=0)
+_DEFAULT_TZINFO = _DEFAULT_DATETIME.tzinfo
```

Comment:
Isn't this just `None`?

BTW the _DEFAULT_DATETIME above can be cdef'd and typed.

Reply:
It might be so in current implementation, but could potentially be changed in the future Python release. I'm basically keeping the original behaviour as much as I can.

Reply:
i think this is more likely to cause confusion than anything else

Reply:
Replaced with explicit `None` in the call.